### ISSUE TITLE
Link to tier description

### ISSUE THREAD

**Issue by hermidalc** [CBIIT/nci-ctd2-dashboard#170]
Can there be a document or paragraph describing what the tiers are

**Comment by hermidalc** [comment]
*Original comment by*: @vdancik

Duplicate of #246.

**Comment by hermidalc** [comment]
*Original comment by*: @vdancik

This was brought up again as Issue #246

**Comment by hermidalc** [comment]
*Original comment by*: @vdancik

I would close this one as done since tier descriptions are in glossary now, but some recent user comments asked for tier information. Maybe there should be a question mark with a link to glossary in places where tiers are used?

**Comment by hermidalc** [comment]
*Original comment by*: @armish

Thanks for the follow-up on this Jessica. This is now assigned back to you. I will hold off from making any changes to the website before I hear from the steering committee, then.

**Comment by hermidalc** [comment]
*Original comment by*: **jessicamazerik**

Update: Daniela would like us to hold off on adding any Tier definitions to the Dashboard. She wants us to revisit this with steering committee in a few months. Arman, you can reassign to me and I will keep it on my radar to bring up in mid-September.

**Comment by hermidalc** [comment]
*Original comment by*: @armish

Thanks Jessica; the text looks great and yes: I can definitely provide this in a pop-up to the user on hover. I am assigning this back to myself and will implement it in the next iteration.

If steering committee wants this information on the homepage, maybe we can re-visit this issue and the related text.

**Comment by hermidalc** [comment]
*Original comment by*: **jessicamazerik**

I'm happy to own this issue. For now, we can pull brief definitions from the Tier Document that is in the process of being finalized by the Steering Committee. Once it is finished, we can also link to the document itself (pending approval from Daniela/NCI and the SC). For now, here are the definitions (from the latest version of the document headings; I will update if they change):

### Data in the Dashboard are categorized as follows: ### 
* Tier 1 - Primary or raw data
* Tier 2 - Confirmation of primary results *in vitro*
* Tier 3 - Validation of results in a cancer-relevant *in vivo* model

Is it possible to make a pop-up window that comes up when you click on the Tier column, where we could place these brief definitions? I think from a user standpoint this would be the most intuitive place to put it. If we want it on the landing page, let me know so I can develop some other text to provide a brief explanation as to why we are defining the Tiers.
Thanks!

**Comment by hermidalc** [comment]
*Original comment by*: @armish

This is a fair comment. We do have text explaining what other terms mean on the home page, but not for tiers themselves. 

Jessica, just assigned this to you. Feel free to re-assign it to somebody else if needed.

Thanks,
